site/src/sphinx/server-annotated-service.rst

```diff
@@ -748,6 +749,38 @@ order commented. It is also the same as the evaluation order of the converters.
                         new MyGlobalExceptionHandler5(),        // order 5
                         new MyGlobalExceptionHandler6());       // order 6
 
+Returning a response
+--------------------
+
+In the earlier examples, the annotated service methods only return :api:`HttpResponse`, however there are
+more response types which can be used in the annotated service.
+
+- :api:`HttpResponse` and :api:`AggregatedHttpMessage`
+
+  - It will be sent to the client without any modification. If an exception is raised while the response is
+    being sent, exception handlers will handle it. If no message has been sent to the client yet,
+    the exception handler can send an :api:`HttpResponse` instead.
+
+- ``Publisher<?>``
```

Comment:
How about: 'Reactive Streams ``Publisher``' ?

Reply:
Thanks! Changed and added a link to https://www.reactive-streams.org/reactive-streams-1.0.2-javadoc/org/reactivestreams/Publisher.html, too.

Reply:
Great!